src/components/sidebar/linkView/LinkView.tsx

```diff
@@ -116,33 +171,34 @@ class LinkView extends React.Component<ILinkViewProps, ILinkViewState> {
                     <div className={s.flexInnerRowFlexEnd}>
                         <InputContainer
                             label='ALKU'
-                            placeholder='1020112'
+                            value={startNode ? startNode.id : '-'}
                         />
                         <Dropdown
                             onChange={this.onChange}
-                            items={['P', 'P1', 'P2']}
-                            selected={'P'}
+                            items={['P', 'X', '-']}
+                            selected={startNode ? startNode.type : 'tyhjä'}
                         />
                         <InputContainer
                             label=''
-                            placeholder='Rautatientori'
+                            value={
+                                startNode && startNode.stop ? startNode.stop!.nameFi : '-'}
```

Comment:
Isn't this in effect forcing empty names to become `-` in the backend.

Reply:
disabled